sql-aggregates/starring-in.sql

```diff
@@ -0,0 +1,9 @@
+select  "genres"."name",
+        count("filmId")
+  from "genres"
+  join "filmGenre" using ("genreId")
+  join "films" using ("filmId")
+  join "castMembers" using ("filmId")
+  join "actors" using ("actorId")
+  where CONCAT("firstName", ' ', "lastName") = 'Lisa Monroe'
```

Comment:
Interesting solution. You could also check `firstName = 'Lisa' and 'lastName = 'Monroe'`